assets/rpip-53/merkle-tree-spec.md

```diff
@@ -0,0 +1,164 @@
+# Specification for the Rewards Merkle Tree
+
+Rocket Pool's [Redstone](https://medium.com/rocket-pool/rocket-pool-the-merge-redstone-601d9efd6b4) update introduced a new rewards system based on a Merkle Tree distribution system.
+For each rewards interval, this tree will capture the RPL rewards (both for node operation and Oracle DAO membership) and the ETH rewards (from the Smoothing Pool, if relevant) for all of the nodes in the Rocket Pool network.
+
+In the [Houston](https://github.com/rocket-pool/docs.rocketpool.net/blob/houston/docs/guides/houston/whats-new.md) update, following [RPIP-53](https://github.com/rocket-pool/RPIPs/blob/rewards_v10/RPIPs/RPIP-53.md) the tree was modified slightly to account for the new [RPL withdrawal address](https://github.com/rocket-pool/RPIPs/blob/main/RPIPs/RPIP-31.md) feature.
```

Comment:
this kind of link seems at risk of going stale - can you do it with a commit hash instead?

and minor: the rpip links could be relative? absolute is also probably fine though

Reply:
I ended up replacing it with a Medium article instead - I used that link in the v2 Beta 1 release so it makes more sense to use that here instead of the docs page.